app/models/forms/bulk_upload_lettings_data_check/soft_errors_valid.rb

```diff
@@ -0,0 +1,40 @@
+module Forms
+  module BulkUploadLettingsDataCheck
+    class SoftErrorsValid
+      include ActiveModel::Model
+      include ActiveModel::Attributes
+      include Rails.application.routes.url_helpers
+
+      attribute :bulk_upload
+      attribute :soft_errors_valid, :string
+
+      validates :soft_errors_valid, presence: true
+
+      def options
+        [
+          OpenStruct.new(id: "yes", name: "Yes, some of these are errors"),
```

Comment:
I don't love yes = errors, no = correct, seems at odds with the "are you sure these values are correct" format we use in the soft validations themselves

Reply:
Hmm, do you mean the content should be different, or how we store it on our side?

Reply:
Okay, I've asked about this as well. Can possibly change later if need to